.github/workflows/main.yaml

```diff
@@ -0,0 +1,47 @@
+name: CI book offensive
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+          php-version:
+            - "8.1"
+    steps:
+      - name: Start PostgreSQL
+        run: |
+            sudo systemctl start postgresql.service
+            pg_isready
+            sudo -u postgres psql -c "create user odm WITH PASSWORD 'odm';"
```

Comment:
You should probably replace `odm` (which is in the name of the other lib) by the default login and password in `.env`, so you will also be able to not set `DATABASE_URL` as the default value will be enough.